### DISCUSSION TITLE
Mac OS Sequoia 15.1.1 - Rust Desk not starting

### DISCUSSION THREAD

**Discussion by github-ps1304** [rustdesk/rustdesk#10015] [Q&A] [answered]
### Bug Description

after updating Mac OS to newest version, Rust Desk Client is not starting anymore; its an company controlled Mac.

### How to Reproduce

install RustDesk Client 1.3.2 on Mac OS Sequoia 15.1.1 or 15.1

### Expected Behavior

Client starting

### Operating system(s) on local (controlling) side and remote (controlled) side

Mac OS 15.1.1

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.2

### Screenshots

<img width="546" alt="image" src="https://github.com/user-attachments/assets/524e3fe4-566c-4876-a7f4-48a8fda8b5b7">

"only Apple Store and known developers are allowed"

**Comment by danielsonfire** [comment]
I am also getting the same issue. Please help

**Comment by rustdesk** [comment, marked as answer]
https://www.google.com/search?q=only+Apple+Store+and+known+developers+are+allowed&rlz=1C5CHFA_enSG1100SG1100&oq=only+Apple+Store+and+known+developers+are+allowed&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQIRigATIHCAIQIRiPAtIBBzc2NGowajeoAgCwAgA&sourceid=chrome&ie=UTF-8
